docs/policy/service-changes-2018.md

```diff
@@ -0,0 +1,188 @@
+If you run a service that queries MyOSG:
+
+!!! info "Action items"
+    - Ensure that the services use <https://my.opensciencegrid.org> instead of <https://myosg.grid.iu.edu>
+    - Let us know what queries you’re making and why you’re making them
+
+[OIM](https://oim.opensciencegrid.org/oim/home) serves as the database for the information used by MyOSG with a web
+interface for data updates.
+The OIM web interface will be retired but we will migrate its data to a series of YAML files held in a GitHub repository.
+After OIM is retired, updates to the aforementioned data can be requested via e-mail or pull-request.
+Documentation forthcoming.
+
+!!! note
+    Please see the [OSG CA](#osg-ca) section for information regarding the OIM certificate service.
+
+The exact dates for retiring the MyOSG and OIM web interfaces will be announced.
+If you experience any problems with the OSG Software repository, please contact us at
+[help@opensciencegrid.org](mailto:help@opensciencegrid.org).
+
+
+RSV
+---
+
+The [central RSV service](https://my.opensciencegrid.org/rgcurrentstatus/index?summary_attrs_showservice=on&summary_attrs_showrsvstatus=on&summary_attrs_showfqdn=on&current_status_attrs_shownc=on&gip_status_attrs_showtestresults=on&downtime_attrs_showpast=&account_type=cumulative_hours&ce_account_type=gip_vo&se_account_type=vo_transfer_volume&bdiitree_type=total_jobs&bdii_object=service&bdii_server=is-osg&start_type=7daysago&start_date=04%2F19%2F2018&end_type=now&end_date=04%2F19%2F2018&all_resources=on&gridtype=on&gridtype_1=on&active_value=1&disable_value=1)
+is a monitoring tool that displays every site service in the OSG.
+It will be retired since there is no longer a need to monitor OSG site status as a whole.
+If you would like to monitor your OSG services, you can access the status page of your local
+[RSV](https://opensciencegrid.github.io/docs/monitoring/install-rsv/) instance.
+
+!!! info "Action item"
+    Before the retirement, you will need to disable the `gratia-consumer` on your local RSV host,
+    which uploads status data to the central RSV service:
+
+        :::console
+        root@server # rsv-control --disable --host <YOUR RSV HOST> gratia-consumer
+        root@server # rsv-control --off --host <YOUR RSV HOST> gratia-consumer
+
+The exact date for retiremnt will be announced.
+If you experience any problems with the central RSV service, please contact us at
+[help@opensciencegrid.org](mailto:help@opensciencegrid.org).
+
+GRACC Accounting
+----------------
+
+No changes are planned for the [GRACC accounting](https://gracc.opensciencegrid.org/dashboard/db/gracc-home?orgId=1)
+service at this time.
+If you experience any problems with GRACC accounting, please contact us at
+[help@opensciencegrid.org](mailto:help@opensciencegrid.org).
+
+OASIS and CVMFS
+---------------
+
+The physical hosting location of the Stratum-0 will be moved but we do not plan any other changes.
```

Comment:
Intro to the services that OSG runs?

Reply:
Also, mention that the move is expected to have no effect on sites. Here and for the Collector.